site/index.ejs

```diff
@@ -32,15 +32,15 @@
     <title><% if (typeof title !== 'undefined') {%><%= title %><% } %></title>
     <% if (typeof cssPath !== 'undefined') { %><link rel="stylesheet" href="<%= cssPath %>" type="text/css" charset="utf-8"><% } %>
     <% if (typeof tracking !== 'undefined' && tracking) { %>
-    <script type="text/javascript" src="https://secure.leadforensics.com/js/77932.js" /><noscript><img src="https://secure.leadforensics.com/77932.png" style="display: none;" /></noscript>
+    <script type="text/javascript" src="https://secure.leadforensics.com/js/77932.js"></script><noscript><img src="https://secure.leadforensics.com/77932.png" style="display: none;" /></noscript>
```

Comment:
Each `<script>` tag must have a separate `</script>` closing tag - they are an exception and cannot self close.